.gitignore

```diff
@@ -17,3 +17,10 @@
 
 /public/artifacts
 !/public/artifacts/.gitkeep
+
```

Comment:
You should move this stuff to your global (in your homedir) .gitignore file.

Reply:
What do you mean?

Reply:
The files you uploaded are not relevant to the project. It is files of your environment. See [detailed explanation](https://thomashunter.name/blog/global-gitignore-vs-repository-gitignore/).